webapp/packages/core-blocks/src/Snackbars/Snackbar.tsx

```diff
@@ -41,42 +42,27 @@ export const Snackbar: React.FC<SnackbarProps> = observer(function Snackbar({
   state,
   time,
 }) {
-  const styles = useStyles(SNACKBAR_COMMON_STYLES);
-  const [mounted, setMounted] = useState(false);
   const translate = useTranslate();
-  const timeStringFromTimestamp = time ? new Date(time).toLocaleTimeString() : '';
-  const translatedTitle = translate(title);
   useStateDelay(closeDelay > 0, closeDelay, onClose);
 
-  useEffect(() => {
-    setMounted(true);
-  }, []);
-
-  return styled(styles)(
-    <notification as="div" {...use({ mounted, closing: !!state?.deleteDelay })}>
-      {type && <NotificationMark type={type} />}
-      <notification-body as="div">
-        <body-text-block as='div'>
-          <text-block-title title={translatedTitle} as='h2'>{translatedTitle}</text-block-title>
-          {message && <message as="div">{translate(message)}</message>}
-        </body-text-block>
-        <notification-footer as='div'>
-          <footer-time as='span'>{timeStringFromTimestamp}</footer-time>
+  return (
+    <SnackbarWrapper closing={!!state?.deleteDelay} closeable onClose={() => onClose(false)}>
```

Comment:
closeable should be inverted, in most cases, we show notification's that's can be closed, so by default snackbar is closable, better to use `persistent`  (or something like) property to indicate that the snackbar can't be closed